crud.py

```diff
@@ -0,0 +1,41 @@
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(models.Author).offset(skip).limit(limit).all()
+
+
+def get_author_by_name(db: Session, name: str):
+    return (
+        db.query(models.Author).filter(models.Author.name == name).first()
+    )
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
```

Comment:
You could include the return type annotation as well here and in the other places